### ISSUE TITLE
Remove bindings

### ISSUE THREAD

**Issue by madoar** [javafxports/openjdk-jfx#304]
Is there a way to remove all bindings done to an `ObservableList`?
The only way I know to remove a binding from an `ObservableList` is by calling `Bindings#unbind...(list1, list2)`.
My problem with this approach is, that I'm often not really interested in saving list2, because it is created on the fly.
Therefore I would like a way to remove **all** bindings from an `ObservableList`, is this possible?

**Comment by KniKnaKnorke** [comment]
Good request. 
Sames goes for function like  .removeAllListeners(); on a ObservableValue.

**Comment by VenatorX** [comment]
I agree, and I think it would be great for cleaning purpose and remove memory leaks to have a removeAllListeners() on all observable.
